api-server/src/server/boot/user.js

```diff
@@ -468,6 +471,93 @@ function postResetProgress(req, res, next) {
   );
 }
 
+function createPostResetProjectProgress(app) {
+  return async function postResetProjectProgress(req, res, next) {
+    const { User } = app.models;
+    const userId = req.user.id;
+    const { blocks } = req.body;
+
+    let toResetChallengeIds = [];
+    let undefinedBlocks = [];
```

Comment:
```suggestion
    const toResetChallengeIds = [];
    const undefinedBlocks = [];
```

Reply:
Thanks for the suggestion, @ShaunSHamilton! I've updated the code as recommended.